src/main/flight/pid.c

```diff
@@ -420,6 +420,16 @@ void pidController(const pidProfile_t *pidProfile, const rollAndPitchTrims_t *an
     // Dynamic ki component to gradually scale back integration when above windup point
     const float dynKi = MIN((1.0f - motorMixRange) * ITermWindupPointInv, 1.0f);
 
+    // rotate old I to the new coordinate system
+    const float gyroToAngle = dT * 2.0f * 3.14159f / 360.0f;
```

Comment:
We have a constant for that. Also, why use 2π and 360 if you could use π and 180?

Reply:
Just writing code as I thought. Makes no difference code size wise. But the constant is nicer, thanks.